src/rgw/rgw_common.h

```diff
@@ -123,6 +123,7 @@ using ceph::crypto::MD5;
 #define RGW_ATTR_RESTORE_TYPE   RGW_ATTR_PREFIX "restore-type"
 #define RGW_ATTR_RESTORE_TIME   RGW_ATTR_PREFIX "restored-at"
 #define RGW_ATTR_RESTORE_EXPIRY_DATE   RGW_ATTR_PREFIX "restore-expiry-date"
+#define RGW_ATTR_TRANSITION_TIME RGW_ATTR_PREFIX "transition-at"
```

Comment:
what is this attr for? i see `write_cloud_tier()` writing it and `restore_obj_from_cloud()` erasing it, but nothing reads/uses it

Reply:
Since it was decided not to change/update mtime for any transition/restore operations, we decided to use a separate attr to record transition time (similar to `restored-at` ).